reconcile/utils/jinja2/utils.py

```diff
@@ -212,6 +213,7 @@ def process_jinja2_template(
         "url": url_makes_sense,
         "s3": lookup_s3_object,
         "flatten_dict": flatten,
+        "yesterday": lambda: (datetime.now() - datetime.timedelta(1)).strftime('%Y-%m-%d'),
```

Comment:
or even fancier:
```suggestion
        "date_in_n_days": lambda n, format='%Y-%m-%d': (datetime.datetime.now() + datetime.timedelta(n)).strftime(format),
```
which works with negative numbers as well :wink:

Reply:
no it has to be yesterday